src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -109,6 +109,11 @@
     $PHPUNIT_VERSION = $MAX_PHPUNIT_VERSION;
 }
 
+if (version_compare($PHPUNIT_VERSION, '10.0', '>=') && version_compare($PHPUNIT_VERSION, '11.0', '<')) {
+    fwrite(STDERR, 'This script does not work with PHPUnit 10.'.\PHP_EOL);
```

Comment:
```suggestion
    fwrite(STDERR, 'The simple-phpunit script does not work with PHPUnit 10.'.\PHP_EOL);
```

Reply:
Not sure about this because when you install the bridge the recipe will create a `bin/phpunit` file which one will run. So no pointer about a `simple-phpunit` script. My guess was that this is also the reason why the check on line 16 also talks about "This script".